### ISSUE TITLE
Possible Enhancement: Make metric names configurable

### ISSUE THREAD

**Issue by moonkev** [RustedBones/akka-http-metrics#58]
Hi @RustedBones - would you be open to a PR that makes the names of metrics configurable?  Possibly add to each individual backend's settings.  For example for the prometheus backend, active requests setting, it could be accessed from the prometheus settings as follows

```scala
  override lazy val active: Gauge = io.prometheus.client.Gauge
    .build()
    .namespace(settings.namespace)
    .name(settings.activeRequestsMetricName) // Could also have a separate MetricsName case class, etc.
```

Where activeRequests requests would default to "requests.active" (as it is now).

If you are open to this, I will happily do for all backends.  Happy to do using another implementation suggestion as well.  We actually have a couple of different use cases where this would be quite beneficial.

**Comment by RustedBones** [comment]
Sure, seems fine!
On the implementation, we can have move the `default` to a companion object in all of the specialized backend.

**Comment by moonkev** [comment]
great, I should be able to have a PR within the next couple of days.

**Comment by RustedBones** [comment]
Fixed by #62 . Thanks a lot!
